src/pg_to_evalscript/javascript_processes/drop_dimension.js

```diff
@@ -0,0 +1,30 @@
+function drop_dimension(arguments) {
+  const { data, name } = arguments;
+
+  if (data === undefined || data === null) {
+    throw new Error("Mandatory argument `data` is either null or not defined.");
+  }
+
+  if (name === undefined || name === null) {
+    throw new Error("Mandatory argument `name` is either null or not defined.");
+  }
+
+  if (typeof name !== "string") {
+    throw new Error("Argument `name` is not a string.");
+  }
+
+  const dim = data.getDimensionByName(name);
+  if (!dim) {
+    throw new Error("A dimension with the specified name does not exist.");
+  }
+
+  if (dim.labels.length > 1) {
```

Comment:
This should be ok in theory, but in practice we don't have labels set most of the time (we do for the bands dimension, but mostly not for others). So perhaps this should be handled better in the `DataCube`, but I think just checking the actual size of the dimension is a good enough solution.

Reply:
Fixed